src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={classNames({ 'is-active': selectedTab === tab })}
```

Comment:
- You compare tabs twice, create a variable and reuse it
- WE suggest you never compare the object with another object with strict comparison. It could produce a lot of bugs. Compare object using their unique identifier (id, title)
![image](https://github.com/mate-academy/react_tabs-js/assets/63243136/d1f7733c-e83e-4263-84bd-650dc1e8be0f)